src/vivarium_gates_bep/globals.py

```diff
@@ -81,45 +169,66 @@ def formatted_location(location):
+                               'treatment_group_{TREATMENT_GROUP}')
+YLDS_COLUMN_TEMPLATE = ('ylds_due_to_{CAUSE_OF_DISABILITY}_among_{MALNOURISHMENT_STATE}_in_age_group_{AGE_GROUP}_'
+                        'treatment_group_{TREATMENT_GROUP}')
+YLLS_COLUMN_TEMPLATE = ('ylls_due_to_{CAUSE_OF_DEATH}_among_{MALNOURISHMENT_STATE}_in_age_group_{AGE_GROUP}_'
+                        'treatment_group_{TREATMENT_GROUP}')
+DALYS_COLUMN_TEMPLATE = ('dalys_due_to_{CAUSE_OF_DEATH}_among_{MALNOURISHMENT_STATE}_in_age_group_{AGE_GROUP}_'
+                         'treatment_group_{TREATMENT_GROUP}')
+DEATHS_COLUMN_TEMPLATE = ('deaths_due_to_{CAUSE_OF_DEATH}_among_{MALNOURISHMENT_STATE}_in_age_group_{AGE_GROUP}_'
+                          'treatment_group_{TREATMENT_GROUP}')
+COUNT_COLUMN_TEMPLATE = ('{COUNT_EVENT}_count_among_{MALNOURISHMENT_STATE}_in_age_group_{AGE_GROUP}_'
+                         'treatment_group_{TREATMENT_GROUP}')
+MEAN_COLUMN_TEMPLATE = ('mean_{METRIC}_among_{MALNOURISHMENT_STATE}_in_age_group_{AGE_GROUP}_'
+                        'treatment_group_{TREATMENT_GROUP}')
+SD_COLUMN_TEMPLATE = ('standard_deviation_{METRIC}_among_{MALNOURISHMENT_STATE}_in_age_group_{AGE_GROUP}_'
+                      'treatment_group_{TREATMENT_GROUP}')
+TRANSITION_COLUMN_TEMPLATE = ('{TRANSITION}_event_count_among_{MALNOURISHMENT_STATE}_in_age_group_{AGE_GROUP}_'
+                              'treatment_group_{TREATMENT_GROUP}')
 
 COLUMN_TEMPLATES = {
     'person_time': PERSON_TIME_COLUMN_TEMPLATE,
     'ylds': YLDS_COLUMN_TEMPLATE,
     'ylls': YLLS_COLUMN_TEMPLATE,
     'dalys': DALYS_COLUMN_TEMPLATE,
+    'deaths': DEATHS_COLUMN_TEMPLATE,
     'counts': COUNT_COLUMN_TEMPLATE,
-    'mortality_counts': MORTALITY_COUNT_COLUMN_TEMPLATE,
-    'means': MEAN_COLUMN_TEMPLATE
+    'means': MEAN_COLUMN_TEMPLATE,
+    'standard_deviations': SD_COLUMN_TEMPLATE,
+    'transitions': TRANSITION_COLUMN_TEMPLATE
 }
 
-TREATMENT_GROUPS = ['no_intervention', 'IFA', 'MMN', 'BEP_plus_MMN']
-CAUSE_OF_DISABILITY = list(CAUSE_MEASURES.keys())
-CAUSE_OF_DEATH = CAUSE_OF_DISABILITY + ['other_causes']
+TREATMENT_GROUPS = ['treated', 'untreated']
+AGE_GROUPS = ['0-1', '1-2']
+CAUSES_OF_DISABILITY = list(CAUSE_MEASURES.keys()) + ['all_causes']
+CAUSES_OF_DEATH = CAUSES_OF_DISABILITY + ['other_causes']
 MALNOURISHMENT_STATES = ['malnourished', 'not_malnourished']
 COUNT_EVENTS = [
-    causes.neonatal_preterm_birth.name,
     LIVE_BIRTH,
     LOW_BIRTH_WEIGHT,
     SHORT_GESTATION,
-    risk_factors.child_stunting.name,
+    f'{risk_factors.child_wasting.name}_cat1',
+    f'{risk_factors.child_wasting.name}_cat2',
+    f'{risk_factors.child_wasting.name}_cat3',
+    f'{risk_factors.child_stunting.name}_cat1',
+    f'{risk_factors.child_stunting.name}_cat2',
+    f'{risk_factors.child_stunting.name}_cat3',
     ALIVE_AT_6_MONTHS
 ]
-METRICS = [LAZ_AT_6_MONTHS, BIRTH_WEIGHT, GESTATIONAL_AGE]
+METRICS = [LAZ_AT_6_MONTHS, WHZ_AT_6_MONTHS, BIRTH_WEIGHT, GESTATIONAL_AGE]
```

Comment:
According to Nicole " we are now referring WHZ as WLZ now for consistency (length instead of height)"

Reply:
Just to be extra annoying, the public health folks might want to call it WLZ until age 1, and then change to calling it WHZ.  The customer is always right?